drivers/sqljs-driver/src/jsMain/kotlin/com/squareup/sqldelight/drivers/sqljs/sqljs.SqlJs.kt

```diff
@@ -0,0 +1,68 @@
+@file:JsQualifier("SqlJs")
+@file:Suppress("INTERFACE_WITH_SUPERCLASS", "OVERRIDING_FINAL_MEMBER", "RETURN_TYPE_MISMATCH_ON_OVERRIDE", "CONFLICTING_OVERLOADS", "EXTERNAL_DELEGATION")
+package com.squareup.sqldelight.drivers.sqljs
+
+import org.khronos.webgl.Uint8Array
+import kotlin.js.*
+
+external interface QueryResults {
+    var columns: Array<String>
+    var values: Array<Array<dynamic /* Number | String | Uint8Array | Nothing? */>>
```

Comment:
how does this work O_O

Reply:
Not sure what you mean
You declare the interface you expect from the JS lib, and then you can use it from Kotlin with type checking etc